bpf/lib/process.h

```diff
@@ -322,6 +324,10 @@ struct binary {
 	char end[STRING_POSTFIX_MAX_LENGTH];
 	// STRING_POSTFIX_MAX_LENGTH reversed last bytes of the path
 	char end_r[STRING_POSTFIX_MAX_LENGTH];
+	// length of the args stored in args
+	__s64 args_length;
```

Comment:
Is this used somewhere?

Reply:
seems not , forgot to remove it when  I decided null terminating was easier.